packages/react-composites/src/composites/localization/locales/en-US/strings.json

```diff
@@ -237,7 +237,7 @@
         "grantedDueToRoleChangeToPresenter": "You're a presenter. You can share content and facilitate the meeting."
       }
     },
-    "surveyQuestion": "How was your call today?",
+    "surveyQuestion": "Help us improve",
```

Comment:
This is less a question now and more a `surveyTitle`

Reply:
Updated